content/docs/community/developers.md

```diff
@@ -56,15 +56,17 @@ The code for the Helm project is organized as follows:
 - Shared libraries are stored in `pkg/`.
 - The `scripts/` directory contains a number of utility scripts. Most of these
   are used by the CI/CD pipeline.
-- The `docs/` folder is used for documentation and examples.
 
-Go dependencies are managed with [Dep](https://github.com/golang/dep) and stored
-in the `vendor/` directory.
+Go dependency management is in flux, and it is likely to change during the course of Helm's lifecycle. We encourage developers to _not_ try to manually manage dependencies. Instead, we suggest relying upon the project's `Makefile` to do that for you. With Helm 3, it is recommended that you are on Go version 1.12 or later.
```

Comment:
can you change this to 1.13? A few tools broke in Go 1.13.

Reply:
```suggestion
Go dependency management is in flux, and it is likely to change during the course of Helm's lifecycle. We encourage developers to _not_ try to manually manage dependencies. Instead, we suggest relying upon the project's `Makefile` to do that for you. With Helm 3, it is recommended that you are on Go version 1.13 or later.
```